src/USDA.NET/Extensions/Serialize.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+using USDA.NET.Food.Report;
+using USDA.NET.Food.Search;
+
+namespace USDA.NET.Extensions
+{
+    public static class SerializeExtension
+    {
+        public static string ToJson(this SearchResult self) => JsonConvert.SerializeObject(self, Converter.Settings);
```

Comment:
If you had to implement ToJson, you should do this on `System.Object` so the implementation is only defined once. However, all JSON operations should really be extracted to the client level (especially via the use of .NET extensions).

Reply:
this function came from https://quicktype.io/ conversion of the api's json. Adding to sys objects seems excessive. Perhaps, the best thing would be to remove it since this API doesn't support posts anyway